boards/arm/imxrt/teensy-4.x/src/imxrt_autoleds.c

```diff
@@ -0,0 +1,165 @@
+/****************************************************************************
+ * boards/arm/imxrt/teensy-4.x/src/imxrt_autoleds.c
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.  The
+ * ASF licenses this file to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the
+ * License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ ****************************************************************************/
+
+/* There are four LED status indicators located on the EVK Board.  The
```

Comment:
There is only the single LED on this board.  I think this comment block needs to be updated.

Reply:
Actually I think there are two of them – user LED and the boot LED (switched on when teensy is in boot mode).